Client/Frontend/Onboarding/ViewControllers/OnboardingDefaultSettingsViewController.swift

```diff
@@ -0,0 +1,255 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Common
+import Shared
+import UIKit
+
+class OnboardingDefaultSettingsViewController: UIViewController, Themeable {
+    private enum UX {
+        static let contentStackViewSpacing: CGFloat = 40
+        static let textStackViewSpacing: CGFloat = 24
+
+        static let titleFontSize: CGFloat = 20
+        static let numeratedTextFontSize: CGFloat = 15
+        static let buttonFontSize: CGFloat = 16
+
+        static let buttonVerticalInset: CGFloat = 12
+        static let buttonHorizontalInset: CGFloat = 16
+        static let buttonCornerRadius: CGFloat = 13
+
+        static let cardShadowHeight: CGFloat = 14
+
+        static let scrollViewVerticalPadding: CGFloat = 30
+        static let topPaddingPhone: CGFloat = 20
+        static let topPaddingPad: CGFloat = 60
+        static let leadingPaddingPhone: CGFloat = 40
+        static let leadingPaddingPad: CGFloat = 200
+        static let trailingPaddingPhone: CGFloat = 40
+        static let trailingPaddingPad: CGFloat = 200
+        static let bottomPaddingPhone: CGFloat = 20
+        static let bottomPaddingPad: CGFloat = 60
+    }
+
+    private lazy var scrollView: UIScrollView = .build { view in
+        view.backgroundColor = .clear
+    }
+
+    lazy var containerView: UIView = .build { view in
+        view.backgroundColor = .clear
+    }
+
+    lazy var contentContainerView: UIView = .build { stack in
+        stack.backgroundColor = .clear
+    }
+
+    private lazy var contentStackView: UIStackView = .build { stack in
+        stack.backgroundColor = .clear
+        stack.alignment = .center
+        stack.distribution = .fill
+        stack.spacing = UX.contentStackViewSpacing
+        stack.axis = .vertical
+    }
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.textAlignment = .center
+        label.numberOfLines = 0
+        label.font = DynamicFontHelper.defaultHelper.preferredBoldFont(withTextStyle: .title3, size: UX.titleFontSize)
+        label.adjustsFontForContentSizeCategory = true
+//        label.accessibilityIdentifier = "\(self.viewModel.infoModel.a11yIdRoot)TitleLabel"
```

Comment:
yeah, the accessibility ID's might have to be coming from the card that presented this card, so you might have to pass the root in. I would, however, add a second identifier to this: `\(root).DefaultBrowserSettings.TitleLabel` or something like that.